opendatahub-io/odh-dashboard
# Check KServe Installation before enabling NIM

## Description


Check KServe installation in the status of `DataScienceCluster` before validating ngc api key, when enabling NIM app.

## How Has This Been Tested?




Negative scenario:
1. Not enable KServe for Openshift AI installation.
2. Enter a valid ngc api key to enable NIM
3. Check if the Job for validating the api key fails
4. Check if the NIM application is not enabled

Positive scenario:
1. Enable KServe for Openshift AI installation.
2. Enter a valid ngc api key to enable NIM
3. Check if the Job for validating the api key succeeds
4. Check if the NIM application is enabled

## Test Impact


No impact on existing code

## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
